src/pages/Deposit.tsx

```diff
@@ -1 +1 @@
-export { default } from 'components/DepositWidget'
+export { default } from 'components/DepositWidget/grid'
```

Comment:
I'm curious as to why create a new grid component replacing the default index but still keeping the index file.

If intending to keep both for loading a different page in different situations, I would suggest to create a new `components/DepositWidget/table` for the existing component and export both from `components/DepositWidget/index`.
Or, if we are gonna have only `grid`, put that instead on `index`

Reply:
in the main description i wrote:
```
TODO:
I still need to remove some test code before approval so draft for now
```
which was referencing my need to replace `index` with `grid` should the style be accepted

Reply:
Oops, guess I should pay closer attention to the PR description =P